src/styles/blocks/main/body.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.page {
```

Comment:
It is recommended to use BEM naming conventions for your classes. 'page' might be too generic and could conflict with other CSS. Consider renaming this class to something more specific and BEM-friendly, e.g., 'main-page'.